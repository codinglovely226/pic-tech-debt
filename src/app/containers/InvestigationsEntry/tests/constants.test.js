import {
  HIDE_INVESTIGATIONS_VIEW_RESULT_MODAL,
  REMOVE_INVESTIGATIONS_VIEW_RESULT_ATTACHMENT,
  RESET_INVESTIGATIONS_VIEW_REQUEST,
  RESET_INVESTIGATIONS_VIEW_RESULT,
  SHOW_INVESTIGATIONS_VIEW_RESULT_MODAL,
  UPDATE_INVESTIGATIONS_REQUESTS,
  UPDATE_INVESTIGATIONS_RESULTS,
  UPDATE_INVESTIGATIONS_VIEW_REQUEST,
  UPDATE_INVESTIGATIONS_VIEW_RESULT_ATTACHMENTS,
  UPDATE_INVESTIGATIONS_VIEW_RESULT_DESCRIPTION,
  UPDATE_INVESTIGATIONS_VIEW_RESULT_NOTES,
} from '../constants';

describe('InvestigationEntry action constants', () => {
  it('HIDE_INVESTIGATIONS_VIEW_RESULT_MODAL', () => {
    expect(HIDE_INVESTIGATIONS_VIEW_RESULT_MODAL).toMatchSnapshot();
  });
  it('REMOVE_INVESTIGATIONS_VIEW_RESULT_ATTACHMENT', () => {
    expect(REMOVE_INVESTIGATIONS_VIEW_RESULT_ATTACHMENT).toMatchSnapshot();
  });
  it('RESET_INVESTIGATIONS_VIEW_REQUEST', () => {
    expect(RESET_INVESTIGATIONS_VIEW_REQUEST).toMatchSnapshot();
  });
  it('RESET_INVESTIGATIONS_VIEW_RESULT', () => {
    expect(RESET_INVESTIGATIONS_VIEW_RESULT).toMatchSnapshot();
  });
  it('SHOW_INVESTIGATIONS_VIEW_RESULT_MODAL', () => {
    expect(SHOW_INVESTIGATIONS_VIEW_RESULT_MODAL).toMatchSnapshot();
  });
  it('UPDATE_INVESTIGATIONS_REQUESTS', () => {
    expect(UPDATE_INVESTIGATIONS_REQUESTS).toMatchSnapshot();
  });
  it('UPDATE_INVESTIGATIONS_RESULTS', () => {
    expect(UPDATE_INVESTIGATIONS_RESULTS).toMatchSnapshot();
  });
  it('UPDATE_INVESTIGATIONS_VIEW_REQUEST', () => {
    expect(UPDATE_INVESTIGATIONS_VIEW_REQUEST).toMatchSnapshot();
  });
  it('UPDATE_INVESTIGATIONS_VIEW_RESULT_ATTACHMENTS', () => {
    expect(UPDATE_INVESTIGATIONS_VIEW_RESULT_ATTACHMENTS).toMatchSnapshot();
  });
  it('UPDATE_INVESTIGATIONS_VIEW_RESULT_DESCRIPTION', () => {
    expect(UPDATE_INVESTIGATIONS_VIEW_RESULT_DESCRIPTION).toMatchSnapshot();
  });
  it('UPDATE_INVESTIGATIONS_VIEW_RESULT_NOTES', () => {
    expect(UPDATE_INVESTIGATIONS_VIEW_RESULT_NOTES).toMatchSnapshot();
  });
});
