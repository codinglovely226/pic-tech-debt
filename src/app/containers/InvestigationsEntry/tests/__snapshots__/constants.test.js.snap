exports[`InvestigationEntry action constants HIDE_INVESTIGATIONS_VIEW_RESULT_MODAL 1`] = `"pic/InvestigationsEntry/HIDE_INVESTIGATIONS_VIEW_RESULT_MODAL"`;

exports[`InvestigationEntry action constants REMOVE_INVESTIGATIONS_VIEW_RESULT_ATTACHMENT 1`] = `"pic/InvestigationsEntry/REMOVE_INVESTIGATIONS_VIEW_RESULT_ATTACHMENT"`;

exports[`InvestigationEntry action constants RESET_INVESTIGATIONS_VIEW_REQUEST 1`] = `"pic/InvestigationsEntry/RESET_INVESTIGATIONS_VIEW_REQUEST"`;

exports[`InvestigationEntry action constants RESET_INVESTIGATIONS_VIEW_RESULT 1`] = `"pic/InvestigationsEntry/RESET_INVESTIGATIONS_VIEW_RESULT"`;

exports[`InvestigationEntry action constants SHOW_INVESTIGATIONS_VIEW_RESULT_MODAL 1`] = `"pic/InvestigationsEntry/SHOW_INVESTIGATIONS_VIEW_RESULT_MODAL"`;

exports[`InvestigationEntry action constants UPDATE_INVESTIGATIONS_REQUESTS 1`] = `"pic/InvestigationsEntry/UPDATE_INVESTIGATIONS_REQUESTS"`;

exports[`InvestigationEntry action constants UPDATE_INVESTIGATIONS_RESULTS 1`] = `"pic/InvestigationsEntry/UPDATE_INVESTIGATIONS_RESULTS"`;

exports[`InvestigationEntry action constants UPDATE_INVESTIGATIONS_VIEW_REQUEST 1`] = `"pic/InvestigationsEntry/UPDATE_INVESTIGATIONS_VIEW_REQUEST"`;

exports[`InvestigationEntry action constants UPDATE_INVESTIGATIONS_VIEW_RESULT_ATTACHMENTS 1`] = `"pic/InvestigationsEntry/UPDATE_INVESTIGATIONS_VIEW_RESULT_ATTACHMENTS"`;

exports[`InvestigationEntry action constants UPDATE_INVESTIGATIONS_VIEW_RESULT_DESCRIPTION 1`] = `"pic/InvestigationsEntry/UPDATE_INVESTIGATIONS_VIEW_RESULT_DESCRIPTION"`;

exports[`InvestigationEntry action constants UPDATE_INVESTIGATIONS_VIEW_RESULT_NOTES 1`] = `"pic/InvestigationsEntry/UPDATE_INVESTIGATIONS_VIEW_RESULT_NOTES"`;
